autoload/cmake4vim.vim

```diff
@@ -161,4 +161,17 @@ function! cmake4vim#SelectBuildType(buildType) abort
 
     silent call cmake4vim#GenerateCMake()
 endfunction
+
+function! cmake4vim#RunTarget() abort
+    if !exists('g:cmake_build_target')
+        echom 'Please select target first!'
+        return
+    endif
+    let s:exec_path = findfile(g:cmake_build_target, utils#cmake#getBuildDir())
+    if strlen(s:exec_path)
```

Comment:
For some platforms we need to add some suffixes (like `.exe` for Windows).

Reply:
Fixed